src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tables = table.children;
+
+for (const part of tables) {
+  const linescollection = part.children;
```

Comment:
The variable name 'linescollection' could be more descriptive. Consider renaming it to 'rows' or 'rowCollection' to better reflect its purpose.